File: java/07 - Demo 3 - Refactoring/src/main/java/extractmethod/Person.java
package extractmethod;

class Person {

	private String name;

	Person(String name) {
		this.name = name;
	}
	
	void printOwing(double amount) {
		printBanner();
		
//		print details
		System.out.print ("name: " + name);
		System.out.print (", amount: " + amount);
	}

	private void printBanner() {
		System.out.print("Person has: ");
	}
}

File: csharp/07 - Demo 3 - Refactoring/Refactoring/ExtractMethod/Person.cs
using System;
using System.Text;

namespace Refactoring.ExtractMethod
{
    public class Person
    {
        private string _name;

        public Person(string name)
        {
            _name = name;
        }

        public void PrintOwing(double amount)
        {
            PrintBanner();

            //		print details
            Console.Write("name: " + _name);
            Console.Write(", amount: " + amount);
        }

        private void PrintBanner() => Console.Write("Person has: ");
    }
}